src/index.ts

```diff
@@ -8,18 +8,18 @@ import { readFileSync } from "fs";
 import { Artifact, ArtifactsResponse } from "./interfaces";
 import { PREFIX } from "./registry";
 import express from 'express';
-import { Logger } from './logger/logger';
+import { Logger, logRequestMiddleware } from './logger/logger';
+import bodyParser from 'body-parser';
```

Comment:
Shouldn't curly braces be used around the import of bodyParser like for Logger above?

Reply:
No, in this specific case I want the object from the body-parser so I can use the following syntax `bodyParser.json()` to use the method  `json()` in from that object.

I would the curly braces import as `I'm getting stuff from **inside** this **place**`. 
I could do `import { json() }` in this specific case btw!

Also I'm removing this package from the import, we can use `express.json()` and it will produce the same effect

Thanks